resources/lang/en/events/death/beach.json

```
@@ -257,63 +255,17 @@
             }
         ]
     },
-    {
-        "event_id": "bch_death_heat_multiapp2",
-        "location": [
-            "beach"
-        ],
-        "season": [
-            "greenleaf",
-            "newleaf",
-            "leaf-fall"
-        ],
-        "tags": [],
-        "weight": 20,
-        "event_text": "r_c noticed m_c hadn't shown up for training again, yet after further prodding realized {PRONOUN/m_c/subject} hadn't been at camp for a few days. However, r_c never expected to find m_c like this: dead and dried out, their tattered pelt lifeless and dull.",
-        "m_c": {
-            "age": [
-                "adolescent"
-            ],
-            "status": [
-                "apprentice"
-            ],
-            "relationship_status": [
-                "app/mentor"
-            ],
-            "dies": true
-        },
-        "r_c": {
-            "age": [
-                "young adult",
-                "adult",
-                "senior adult"
-            ],
-            "status": [
-                "any"
-            ]
-        },
-        "history": [
-            {
-                "cats": [
-                    "m_c"
-                ],
-                "reg_death": "m_c died due to dehydration and overheating."
-            }
-        ]
-    },
     {
         "event_id": "bch_death_heat_multiapp3",
         "location": [
             "beach"
         ],
         "season": [
-            "greenleaf",
-            "newleaf",
-            "leaf-fall"
+            "greenleaf"
         ],
         "tags": [],
         "weight": 20,
-        "event_text": "r_c realizes {PRONOUN/r_c/subject} {VERB/r_c/haven't/hasn't} seen m_c around camp for a few days. Checking {PRONOUN/m_c/poss} favorite spots yields nothing, so r_c decides to go out and search for {PRONOUN/m_c/object} in the Clan's territory. However, r_c could never have expected to find m_c's shriveled-up body, clearly dead for some time now.",
+        "event_text": "r_c discovered m_c's dead body. It seems {PRONOUN/m_c/subject} died to dehydration and heat.",
```

```suggestion
        "event_text": "r_c discovered m_c's dead body. It seems {PRONOUN/m_c/subject} succumbed to dehydration and heat.",
```
personally don't think people die "to" dehydration, they die "of" it. i recall this discussion before tho so am happy to be overruled in the name of Freedom (americanisms vs bri'ishisms)